<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LogiManage</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1">
    <property name="leftMargin">
     <number>12</number>
    </property>
    <property name="topMargin">
     <number>12</number>
    </property>
    <property name="rightMargin">
     <number>12</number>
    </property>
    <property name="bottomMargin">
     <number>12</number>
    </property>
    <item>
     <widget class="QFrame" name="TopBar">
      <property name="frameShape">
       <enum>QFrame::Shape::StyledPanel</enum>
      </property>
      <layout class="QHBoxLayout" name="topBarLayout">
       <property name="leftMargin">
        <number>14</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>14</number>
       </property>
       <property name="bottomMargin">
        <number>10</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="logoWrap">
         <item>
          <widget class="QLabel" name="Logo">
           <property name="minimumSize">
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="text">
            <string>L</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignmentFlag::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AppName">
           <property name="styleSheet">
            <string>font-weight:700; font-size: 14pt;</string>
           </property>
           <property name="text">
            <string>LogiManage</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="navTabs">
         <item>
          <widget class="QToolButton" name="btnDashboard">
           <property name="text">
            <string>Dashboard</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnOrders">
           <property name="text">
            <string>Orders</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnClients">
           <property name="text">
            <string>Clients</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnReports">
           <property name="text">
            <string>Reports</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rightControls">
         <item>
          <widget class="QLineEdit" name="SearchBox">
           <property name="placeholderText">
            <string>Searchâ€¦</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnTheme">
           <property name="toolTip">
            <string>Toggle theme</string>
           </property>
           <property name="text">
            <string>ðŸŒ“</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="btnBell">
           <property name="text">
            <string>ðŸ””</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btnProfile">
           <property name="text">
            <string>Jane Doe</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stacked">
      <widget class="QWidget" name="pageDashboard"/>
      <widget class="QWidget" name="pageOrders"/>
      <widget class="QWidget" name="pageClients"/>
      <widget class="QWidget" name="pageSettings"/>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
